packages/core/src/components/Accordion/Item.tsx

```diff
@@ -76,6 +83,10 @@ export default withStyles(
       padding: `${unit}px ${unit * 2}px ${unit * 2}px`,
     },
 
+    body_noSpacing: {
+      padding: 0,
```

Comment:
let's keep the spacing on the bottom. that's how the request for this prop originally looked from danks:

![image](https://user-images.githubusercontent.com/306275/65263839-3806ef80-dac2-11e9-8fb8-750da9d089a5.png)

```jsx
padding: `0 0 ${unit * 2}px`,
```

Reply:
Will do!

Reply:
Done!